Sources/StreamChatUI/ChatMessageList/TypingIndicatorView.swift

```diff
@@ -47,7 +47,7 @@ open class _TypingIndicatorView<ExtraData: ExtraDataTypes>: _View, UIConfigProvi
         componentContainerView.alignment = .center
         // UIStackView.spacingUseSystem returns 0, because we are not really in stackView in here,
         // so let's use `directionalLayoutMargins`
-        componentContainerView.spacing = directionalLayoutMargins.trailing
+        componentContainerView.spacing = .init(rawValue: directionalLayoutMargins.trailing)
```

Comment:
btw I don't think this line is actually needed now

Reply:
Yes, `auto` should be enough right?

Reply:
Yep